lib/content-services/src/lib/search/components/search-filter-tabbed/search-filter-tabbed.component.scss

```diff
@@ -1,20 +1,5 @@
 adf-search-filter-tabbed {
-    .mat-tab-label {
-        flex: 1;
-
-        &.mat-tab-label-active {
-            border-bottom: 2px solid var(--theme-primary-color);
-        }
-    }
-
-    // The important tag is used here as a workaround for a bug in angular material, when MatTabs are used in conjunction with MatMenu
```

Comment:
Those classes are applied anyway on those elements, so there is no need to add them again